docs/product-spec.md

```diff
@@ -0,0 +1,104 @@
+# VxMail Product Spec
+
+## Background & Motivation
+
+The COVID-19 pandemic presents a significant challenge to the US
+General Election of 2020. How can we safely hold elections when it may
```

Comment:
```suggestion
The COVID-19 pandemic presents a significant challenge to the 2020
General Election in the United States. How can we safely hold elections when it may
```